packages/config/tsconfig/tsconfig.app.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://schemastore.org/tsconfig.json",
+  "extends": "@vue/tsconfig/tsconfig.dom.json",
+  "compilerOptions": {
+    // strictness options - temporarily disabled for easier migration
```

Comment:
I really don't like the way typescript accepts comments like this, but 🤷 I suppose